documentation/book/proc-configuring-external-clients-to-trust-cluster-ca.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='configuring-external-clients-to-trust-cluster-ca-{context}']
+= Configuring external clients to trust the cluster CA 
+
+This procedure will show you how to configure a Kafka client that's outside the {ProductPlatformName} cluster (that is, connecting to the `external` listener on port 9094) so that it trusts the cluster CA certificate.
+
+This procedure can also be used for clients within {ProductPlatformName} connecting to the `tls` listener on port 9093, but it is usually more convenient to access the `Secrets` using a volume mount in the client `Pod`.
+
+This procedure needs to be followed both initially, when setting up the client, and also during the renewal period, when the old clients CA certificate is replaced with a new one.
+
+IMPORTANT: The `_<cluster-name>_-cluster-ca-cert` `Secret` will contain more than one CA certificate during CA certificate renewal. Clients should add _all_ of them to their truststores.
+
+.Prerequisites
+
+* The Cluster Operator up and running
+* An `Kafka` resource within the {ProductPlatformName} cluster.
+* A Kafka client application outside the {ProductPlatformName} cluster which will connect using TLS and needs to trust the cluster CA certificate.
+
+.Procedure
+
+. The cluster CA certificates can be extracted from the generated `_<cluster-name>_-cluster-ca-cert` `Secret`.
+ifdef::Kubernetes[]
++
+On {KubernetesName}, the certificates can be extracted with the following command:
```

Comment:
_Please reword as follows:_

On {KubernetesName}, run the following command to extract the certificates: